html/themes/custom/common_design_subtheme/components/rw-entity-meta/rw-entity-meta.css

```diff
@@ -161,26 +161,17 @@
   background: #f65c51;
 }
 
+.taxonomy-term--disaster--full dl.rw-entity-meta dd.rw-entity-meta__tag-value--status--past:before {
```

Comment:
@orakili Can you determine why the Disaster status isn't being applied here?
The `<dl>` gets a class of `rw-entity-meta__tag-value--status--ongoing` even though it should be `past`
Then this rule would work.
Example:
![Uploading Selection_024.png…]()

Reply:
@left23 The disaster status entity meta class in `taxonomy-term--disaster--full.html.twig` was hardcoded to `ongoing`. Fixed in https://github.com/UN-OCHA/rwint9-site/pull/28/commits/a214750e8b77ad8d633c3a54f3d10a7e1abbb948.